lib/utils/getCertificate.js

```diff
@@ -34,6 +39,7 @@ function getCertificate(logger) {
     const attributes = [{ name: 'commonName', value: 'localhost' }];
     const pems = createCertificate(attributes);
 
+    mkdirp.sync(certificateDir);
```

Comment:
https://nodejs.org/api/fs.html#fs_fs_mkdir_path_options_callback
```
fs.mkdirSync(path, { recursive: true })
```

minimum suppored version Node.js is `10.13` for the next major release

Reply:
Ah, missed that. Wonderful! Will fix today.

Reply:
Done!